### ISSUE TITLE
Compatibility Tables on symphonyextensions.com

### ISSUE THREAD

**Issue by twiro** [symphonists/symphony-extensions-network#9]
Just noticed that older minor releases now get bundled as `2.2.x`, `2.3.x`, `2.4.x`etc. in the compatibility table on symphonyextensions.com.

While it's a good move in general to clean & tighten up the tables I think the current solution is somehow saving at the wrong end:

While almost every extension that's compatible with 2.6.0 will also be compatible with 2.6.7 and every 2.6.x-release in-between the same is not true for compatibility of "patch"-releases in-between older minor releases (before the introduction of semver) .

For example: Symphony 2.3.3 came with a [breaking change](https://github.com/symphonycms/symphony-2/issues/1719) regarding file upload paths. Thus all upload extensions needed to treat file uploading and storing different in Symphony 2.3.3 than they did in 2.3.2.  
I just resubmitted the [Reflected File Upload](http://symphonyextensions.com/extensions/reflecteduploadfield/) extension and the compat-table on symphonyextensions now shows version 1.1 as compatible with Symphony 2.3.x - but this is [only true for Symphony 2.3.0, 2.3.1 and 2.3.2](https://github.com/twiro/reflecteduploadfield/blob/1.2/extension.meta.xml#L21-L29) - as of Symphony 2.3.3 version 1.1 of the extension will break things as Symphony 2.3.3 requires version 1.2 of the extension due to the above mentioned change regarding the file upload path.

I still have some old sites running on Symphony 2.2.x and 2.3.x (I know, better should update) and the detailed compat-tables on symphonyextensions always were of great help when updating 
to a newer Symphony version and fighting the fact that **every single extension might break with every single patch-release**.

So I guess these detailed compat-tables for older symphony releases (at least 2.2.x and 2.3x) are still pretty useful for every developer who needs to update these installations to a newer version.
### tl;dr

I would vote for bundling patch-versions of all minor releases starting with Symphony 2.5.0 (the introduction of semver) and keeping single patch-versions listed for older Symphony releases that were not following semantiv versioning (at least for 2.3.x, better also for 2.2.x).

So the latest parts of the compat-table would look something like that:

| 2.2.5 | 2.3.0 | 2.3.1 | 2.3.2 | 2.3.3 | 2.3.4 | 2.3.5 | 2.3.6 | 2.4.0 | 2.5.x | 2.6.x |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| ... | ... | ... | ... | ... | ... | ... | ... | ... | ... | ... |

**Note:** This proposal is meant for the [single extension pages](http://symphonyextensions.com/extensions/reflecteduploadfield/) that offer more space for a detailled compat-table than the [compatibility matrix](http://symphonyextensions.com/matrix/) does. In the matrix (that's mainly used to get a quick overview) a general bundling of all patch-versions (just listing `2.2.x`, `2.3.x`, `2.4.x`, `2.5.x` and `2.6.x`) would be totally sufficient in my eyes.

**Comment by nitriques** [comment]
I was think about this:

| 2.2.x | 2.3.x | 2.4.0 | 2.5.x | 2.6.x |
| --- | --- | --- | --- | --- |
| 2.2.0... | 2.3.0... | ... | ... | ... |
| 2.2.1... | 2.3.1... | ... | ... | ... |
| 2.2.2... | 2.3.2... | ... | ... | ... |

?
